src/main/java/net/minecraftforge/registries/ForgeRegistry.java

```diff
@@ -839,6 +850,42 @@ public Snapshot makeSnapshot()
         return ret;
     }
 
+    public class RegistryCodec implements Codec<V> {
```

Comment:
Is there a reason this implementation detail is public instead of private?

Reply:
Not at all, it can (and should probably) be private.

Reply:
done